src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostExprTyping.scala

```diff
@@ -97,11 +97,13 @@ trait GhostExprTyping extends BaseTyping { this: TypeInfoImpl =>
       }
       permWellDef ++ expWellDef
 
-    case n: PPredicateAccess => resolve(n.pred) match {
-      case Some(_: ap.PredicateCall) => noMessages
-      case Some(_: ap.PredExprInstance) => noMessages
-      case _ => error(n, s"expected reference, dereference, field selection, or predicate expression instance, but got ${n.pred}")
-    }
+    case n: PPredicateAccess =>
```

Comment:
What is a predicate access (in particular compared to a predicate call and predicate expression instance)? Also, as above, we do not have a test case exercising this error.

Reply:
Should we add a test for "Cannot reveal predicate access"? If yes, what should it look like?